architecture-decisions/0001-record-architecture-decisions.md

```diff
@@ -0,0 +1,19 @@
+# 1. Record architecture decisions
+
+Date: 2020-11-16
+
+## Status
+
+Accepted
+
+## Context
+
+As we take steps to modernize CI and dependency updating, @gr2m and @travi have taken steps to discuss trade-offs of various options. During these discussions, the value of capturing the decisions from these discussions came up. Interest from both of us in ADRs was mentioned.
```

Comment:
```suggestion
As we take steps to modernize CI and dependency updating, [@gr2m](https://github.com/gr2m/) and [@travi](https://github.com/travi/) have taken steps to discuss trade-offs of various options. During these discussions, the value of capturing the decisions from these discussions came up. Interest from both of us in ADRs was mentioned.
```

Reply:
i guess ive been spoiled by https://github.com/remarkjs/remark-github and forgot that doesnt work by default